ranvier

```diff
@@ -13,9 +13,9 @@
 
 /* NPM Modules */
 const semver = require('semver');
-const net = require('net');
+//const net = require('net'); Not Being used anywhere in the code
 const commander = require('commander');
-const argv = require('optimist').argv;
+//const argv = require('optimist').argv;   New Node provides argv functuion built in
```

Comment:
Good catch, though it's preferred to just delete these lines rather than comment them out.

Reply:
I just wanted to be sure as I am a beginner so i didn't.